### PR TITLE
Resolved changes in merge conflicts

### PR DESCRIPTION
**Pull request by nforman91**
## Description

I wanted to reconfigure the family page to the same profile selection format as the [mockup](https://xd.adobe.com/view/a7c39f50-7d8e-4d69-8074-e9d6ebc2e8e3-f290/screen/4031d736-0547-4cdb-b473-7ce6d680c67a?fullscreen). The page needs to be hooked up to state by a frontend engineer.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/ParentFamily/ParentFamilyHome.js [hunk 1]
```diff
@@ -36,7 +33,27 @@ const ParentFamilyHome = () => {
...
+            <button className="profile-select-button">VIEW ACCOUNT</button>
+          </div>
+        </div>
+        <div className="profile-select-logout-container">
+          <button className="family-page-logout-button">
+            LOG OUT OF SHARED ACCOUNT
+          </button>
+        </div>
+        {/* <div className="profile-card-container">
           <div className="profile-details-headers">
```
Comment (Ryan-E-Mark) [inline src/components/pages/ParentFamily/ParentFamilyHome.js:57]: Don't forget to remove commented out code, unless there is a reason for keeping it that I'm missing.

**Review by Ryan-E-Mark — APPROVED** [review]
Great way to reconfigure the family page to follow the stakeholder's vision. This is an important step in getting the account management working, thanks for putting this together!

**Review by ElchinNasirov — APPROVED** [review]
Looks great. Good implementation.

File: src/components/pages/ParentFamily/ParentFamilyHome.js (hunk 1 above)
Comment (JesusNavarrete215) [inline src/components/pages/ParentFamily/ParentFamilyHome.js:57]: Other else than removing the commented out code, everything else is looking good.

**Pull request merged**